config/quality.gradle

```diff
@@ -49,7 +49,7 @@ task checkstyleTest(type: Checkstyle) {
     classpath = files()
 }
 
-task findbugs(type: FindBugs) {
+task findbugs(type: FindBugs, dependsOn: ['compileDebugSources', 'compileReleaseSources']) {
```

Comment:
Does this ensure that the project is built/compliled every time before findbugs checks are applied?